easybuild/easyconfigs/p/PythonPlus/PythonPlus-3.7.4-foss-2018b-v20.02.1.eb

```diff
@@ -7,15 +7,17 @@ versionsuffix = '-v20.02.1'   # In format YY.MM.IncrementedReleaseNumber.
 homepage = 'https://www.python.org/'
 description = """The PythonPlus bundle contains a plain vanilla Python supplemented with a selection of add-on modules."""
 
-#
-# numpy requires BLAS -> use full foss toolchain as opposed to GCCcore.
 # scipy requires compiler option -fPIC.
-#
 toolchain = {'name': 'foss', 'version': '2018b'}
 toolchainopts = {'pic': True}
 
+exts_defaultclass = 'PythonPackage'
+# overwrite default sanity check
+exts_filter = ('python -c "import scipy"', "")
```

Comment:
This does not make sense to me: for every Python package it will now check if `scipy` can be imported. That is Ok for checking whether `scipy` was installed correctly, but not for the 197 other packages.

Reply:
Feel free to suggest another `exts_filter` value (that doesn't need to constructed by hand from the requirements.txt ;-) ).